### PR TITLE
don't raise an exception if there is an error

### PR DESCRIPTION
**Pull request by pdmars**
allow it to poll until timeout

### CODE REVIEW

File: reddwarf/tests/fakes/nova.py [hunk 1]
```diff
@@ -114,7 +114,12 @@ def confirm_resize(self):
 
     def delete(self):
         self.schedule_status = []
-        self._current_status = "SHUTDOWN"
+        # TODO(pdmars): This is less than ideal, but a quick way to force it
```
Comment (ed-) [inline reddwarf/tests/fakes/nova.py:117]: I totally do exactly the same thing in create

**Comment by ed-** [comment]
+1

File: reddwarf/taskmanager/models.py [hunk 2]
```diff
@@ -213,7 +213,6 @@ def server_is_finished():
                     msg = "Server %s got into ERROR status during delete " \
                           "of instance %s!" % (server.id, self.id)
                     LOG.error(msg)
-                    raise ReddwarfError(msg)
```
Comment (rnirmal) [inline reddwarf/taskmanager/models.py:216]: so we are just removing the raise here? This is fine, we'll just have to follow thru and make sure there's no dangling instances.

**Comment by rnirmal** [comment]
+1

File: reddwarf/taskmanager/models.py (hunk 2 above)
Comment (TimSimpsonR) [inline reddwarf/taskmanager/models.py:216]: We still raise on time out, so thats something.

**Comment by TimSimpsonR** [comment]
+1

**Comment by cp16net** [comment]
+1

**Pull request merged**